### ISSUE TITLE
iadmin throwing "ERROR: rcGeneralAdmin failed with error -850000 CAT_PASSWORD_ENCODING_ERROR"

### ISSUE THREAD

**Issue by jerry57** [irods/irods#3138]
When trying to set/change a user's password as a valid iRODS admin user we are seeing this error

issue the following command replacing "<validPassword>" with a real password
iadmin moduser qa-test4 password <validPassword>

Here is what is returned
ERROR: rcGeneralAdmin failed with error -850000 CAT_PASSWORD_ENCODING_ERROR 
Level 0: Error with password encoding.  This can be caused by not connecting directly to the ICAT host, not using password authentication (using GSI or Kerberos instead), or entering your password incorrectly (if prompted).

Expected behaviour is to update/set the users password as long as user executing command is a valid iRODS admin.

**Comment by trel** [comment]
Are you connected to the iCAT server directly with `iadmin`?

**Comment by jerry57** [comment]
not really sure what you mean by connected directly, I am on a remote machine authenticated as an iRODS admin issuing the command.  I am guessing No is the answer you are looking for though

Is this only supposed to work when "directly connected"?

**Comment by trel** [comment]
Yes, to change the password of a user, the administrator must be connected directly to the server with access to the database (not a 'resource' server).  This is a known limitation of the existing protocol.

`This can be caused by not connecting directly to the ICAT host`

**Comment by trel** [comment]
When I say 'directly connected', I mean that the client machine running `iadmin` needs to have the iCAT server listed as the `irods_host` in the `~/.irods/irods_environment.json`.

**Comment by alanking** [comment]
While this log message still exists, I cannot reproduce this problem in the latest iRODS releases. I'm going to close this. Please open a new issue with steps to reproduce if it persists.
